config/stored_requests.go

```diff
@@ -318,89 +351,24 @@ type PostgresUpdatePolling struct {
 	AmpQuery string `mapstructure:"amp_query"`
 }
 
-func (cfg *PostgresUpdatePolling) validate(dataType DataType, errs []error) []error {
+func (cfg *DatabaseUpdatePolling) validate(dataType DataType, errs []error) []error {
 	section := dataType.Section()
 	if cfg.Query == "" {
 		return errs
 	}
 
 	if cfg.RefreshRate <= 0 {
-		errs = append(errs, fmt.Errorf("%s: postgres.poll_for_updates.refresh_rate_seconds must be > 0", section))
+		errs = append(errs, fmt.Errorf("%s: database.poll_for_updates.refresh_rate_seconds must be > 0", section))
 	}
 
 	if cfg.Timeout <= 0 {
-		errs = append(errs, fmt.Errorf("%s: postgres.poll_for_updates.timeout_ms must be > 0", section))
+		errs = append(errs, fmt.Errorf("%s: database.poll_for_updates.timeout_ms must be > 0", section))
 	}
-
-	if !strings.Contains(cfg.Query, "$1") || strings.Contains(cfg.Query, "$2") {
-		errs = append(errs, fmt.Errorf("%s: postgres.poll_for_updates.query must contain exactly one wildcard", section))
+	if !strings.Contains(cfg.Query, "$LAST_UPDATED") {
+		errs = append(errs, fmt.Errorf("%s: database.poll_for_updates.query must contain $LAST_UPDATED parameter", section))
 	}
```

Comment:
I'll try to come back with some suggestions on this shortly. It would be preferable if this was easy to remove as we don't anticipate supporting the old style indefinitely.

Reply:
I've implemented a mechanism to migrate query parameters and the migration of some parameters that I'm confident migrates properly.: https://github.com/prebid/prebid-server/pull/2356/commits/5afcc5ecf57d8d054c77ef16f214f3583ae8ef5c
Please provide me with missing parameters that also need to migrate here.